aicsimageio/aics_image.py

```diff
@@ -333,19 +333,31 @@ def get_image_dask_data(
             Default: The current image dimensions. i.e. `self.dims`
 
         kwargs:
-            C=1: specifies Channel 1
-            T=3: specifies the fourth index in T
-            D=n: D is Dimension letter and n is the index desired D should not be
-            present in the out_orientation
+            * C=1: specifies Channel 1
+            * T=3: specifies the fourth index in T
+            * D=n: D is Dimension letter and n is the index desired. D should not be
+              present in the out_orientation.
+            * D=[a, b, c]: D is Dimension letter and a, b, c is the list of indicies
+              desired. D should be present in the out_orientation.
+            * D=(a, b, c): D is Dimension letter and a, b, c is the tuple of indicies
+              desired. D should be present in the out_orientation.
+            * D=range(...): D is Dimension letter and range is the standard Python
+              range function. D should be present in the out_orientation.
+            * D=slice(...): D is Dimension letter and slice is the standard Python
+              slice function. D should be present in the out_orientation.
 
         Returns
         -------
         data: dask array
             The read data with the dimension ordering that was specified with
             out_orientation.
 
-        Note: If a requested dimension is not present in the data the dimension is
-        added with a depth of 1.
+        Notes
+        -----
+        * If a requested dimension is not present in the data the dimension is
+          added with a depth of 1.
+
+        See `aicsimageio.transforms.reshape_data` for more details.
```

Comment:
Your documentation is great. A code block example might enhance this even further.

Reply:
Added examples to all functions